.pre-commit-config.yaml

```
@@ -11,31 +11,20 @@ ci:
     submodules: false
 
 repos:
--   repo: https://github.com/python-poetry/poetry
-    rev: ''  # add version here
+-   repo: local
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)